deep_research/configs/enhanced_research.yaml

```
@@ -63,9 +63,22 @@ steps:
 settings:
   docker:
     requirements:
-      - openai>=1.0.0
+      - zenml>=0.83.1
       - tavily-python>=0.2.8
+      - exa-py>=1.0.0
       - PyYAML>=6.0
       - click>=8.0.0
       - pydantic>=2.0.0
       - typing_extensions>=4.0.0
+      - requests
+      - anthropic>=0.52.2
+      - litellm==1.69.1
+      - langfuse==2.60.8
+    environment:
+      OPENROUTER_API_KEY: "YOUR_OPENROUTER_API_KEY"
+      TAVILY_API_KEY: "YOUR_TAVILY_API_KEY"
```

All of these should be picked up from the env variables to make this a lot easier 

```suggestion
      OPENROUTER_API_KEY: ${OPENROUTER_API_KEY}
      TAVILY_API_KEY: ${TAVILY_API_KEY}
```

---

Fixed